eos-firewall-localonly

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash -e
```

Comment:
Maybe, but it's really not that complex - just some loops and some variable substitution. There was nothing that made me think twice about how to do it in shell apart from the variable indirect lookup, but that was forced upon me by NetworkManager. :P

If the script had some python oneliners in it, we could very easily reach agreement here. :)